DeepCloner/DeepCloner.csproj

```diff
@@ -1,13 +1,13 @@
 <Project Sdk="Microsoft.NET.Sdk">
 	<PropertyGroup>
-		<TargetFramework>net472</TargetFramework>
+		<TargetFramework>netstandard2.0</TargetFramework>
 		<GeneratePackageOnBuild>true</GeneratePackageOnBuild>
 		<IncludeSource>true</IncludeSource>
 		<IncludeSymbols>true</IncludeSymbols>
 		
 		<AssemblyTitle>EVRY.DeepCloner</AssemblyTitle>
 		<Description>Library for cloning objects</Description>
-		<Version>0.10.2.0</Version>
+		<Version>0.10.3.0</Version>
```

Comment:
In this case I think it's better to update second version to <Version>0.11.0.0</Version>

Reply:
done.